test/test.js

```diff
@@ -0,0 +1,124 @@
+var assert = require('assert');
+const t = require('../index');
+const {
+    FlightPriceWatcher
+} = require('../index');
+
+// /!\ Go to the botton and add the API KEY before testing /!\
+
+describe('FlightPriceWatcher', () => {
+    describe('#constructor()', function () {
+        it('should construct', function () {
+            let app = new FlightPriceWatcher();
+            assert.ok(app);
+        });
+    });
+    describe('#run()', () => {
+        // Check if the keyboard key is the right color when none is stored
+        // Means getLastPrice returns null
+        it('Can get price when none is stored', function () {
+            let app = new FlightPriceWatcher();
+            assert.ok(app.setLastPrice(null));
+            assert.equal(app.getLastPrice(), null);
+          })
+        it('check the color of the key', async function () {
```

Comment:
This test is not clear.
You can use a mock to simulate the response of the method getPrice. There is example in the travis applet  I think